Dockerfile

```diff
@@ -40,6 +40,8 @@ RUN --mount=target=. <<EOT
 EOT
 
 FROM base as build
+# enable scanning for this stage
+ARG BUILDKIT_SBOM_SCAN_STAGE=true
```

Comment:
We don't need scanning at this step - the golang dependencies are automatically loaded from the go binary `/bin/syft-scanner`. This just also pulls in all the stuff from the build stages.

This doesn't work in buildkit, because syft doesn't open up tarballs it seems, I'll see if I can follow that up.